lib/dogapi/v1/monitor.rb

```diff
@@ -17,9 +17,13 @@ def monitor(type, query, options = {})
       end
 
       def update_monitor(monitor_id, query, options)
-        body = {
-          'query' => query,
-        }.merge options
+        body = {}.merge options
```

Comment:
Rather than merging an empty hash, I would write it as :

```ruby
      def update_monitor(monitor_id, query, options)
        unless query.nil?
          options['query'] = query
          warn '[DEPRECATION] query param is no longer required for update'
        end

        request(Net::HTTP::Put, "/api/#{API_VERSION}/monitor/#{monitor_id}", nil, options, true)
      end
```

Reply:
For some reason it's being picky about the order and local tests don't pass with this 
 ¯\_(ツ)_/¯